charts/newrelic-k8s-metrics-adapter/tests/job_patch_job_patchapiservice_test.yaml

```diff
@@ -0,0 +1,69 @@
+suite: test naming helper for job-patchAPIService
+templates:
+  - templates/apiservice/job-patch/job-patchAPIService.yaml
+release:
+  name: my-release
+  namespace: my-namespace
+tests:
+  - it: spec metadata name is is correctly defined
+    asserts:
+      - matchRegex:
+          path: spec.template.metadata.name
+          pattern: ^.*-apiservice-patch
+  - it: container args are correctly defined
+    asserts:
+      - matchRegex:
+          path: spec.template.spec.containers[0].args[2]
+          pattern: ^--secret-name=.*-apiservice
+
+  - it: serviceAccountName is correctly defined
+    asserts:
+      - matchRegex:
+          path: spec.template.spec.serviceAccountName
+          pattern: ^.*-apiservice
+  - it: has the correct image
+    set:
+      personalAPIKey: 21321
+      apiServicePatchJob:
+        image:
+          repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
+          tag: "latest"
+    asserts:
+      - matchRegex:
+          path: spec.template.spec.containers[0].image
+          pattern: ^.*k8s.gcr.io/ingress-nginx/kube-webhook-certgen:latest
+  - it: renders when extraEnv, extraEnvFrom, extraVolumes and extraVolumeMounts are set
+    set:
+      personalAPIKey: 21321
+      apiServicePatchJob:
+        extraEnv:
+          - name: ENV_VAR1
+            value: "var1"
+          - name: ENV_VAR2
+            value: "var2"
+        extraEnvFrom:
+          - configMapRef:
+              name: special-config
+        extraVolumes:
+          - name: tmpfs-data
+            emptyDir: {}
+        extraVolumeMounts:
+          - mountPath: /var/db/newrelic-infra/data
+            name: tmpfs-data
+    asserts:
+      - hasDocuments:
+          count: 1
+  - it: tolerations are correctly rendered
```

Comment:
Tolerations come [from the common-library](https://github.com/newrelic/helm-charts/blob/master/library/CHART-TEMPLATE/tests/common_library_tolerations_test.yaml) and [they are tested there](https://github.com/newrelic/helm-charts/blob/master/library/CHART-TEMPLATE/tests/common_library_tolerations_test.yaml).

Reply:
yes, but here it's just tested that they get rendered.

Reply:
If the template is having an issue they would not get rendered.

Reply:
I've modified it to not be specifically tested, but just added to the vaules